src/main/java/com/puppycrawl/tools/checkstyle/checks/SuppressWarningsHolder.java

```diff
@@ -437,6 +439,8 @@ private static DetailAST getAcceptableParent(DetailAST child) {
             case TokenTypes.TYPE_ARGUMENT:
             case TokenTypes.IMPLEMENTS_CLAUSE:
             case TokenTypes.DOT:
+            case TokenTypes.ANNOTATION:
+            case TokenTypes.MODIFIERS:
```

Comment:
Can you explain the addition of the modifiers. Annotation seems to make sense, but I don't understand modifiers or member value pair.

Reply:
In this example

    @Getter(onMethod_ = @SuppressWarnings("unchecked")) 
    private final List<String> problematicProperty = List.of();

Annotation Target is ANNOTATION_MEMBER_VALUE_PAIR and Parent is ANNOTATION

And in this example

    @CustomAnnotation(@SuppressWarnings("unchecked")) 
    private List<String> nonLombokProblematicField;

    @Retention(RetentionPolicy.RUNTIME)
    @Target(ElementType.FIELD)
    public @interface CustomAnnotation {
        SuppressWarnings value();
    }
Annotation Target is ANNOTATION and Parent is MODIFIERS